guides/source/active_record_querying.md

```diff
@@ -1266,6 +1266,9 @@ class Client < ActiveRecord::Base
 end
 ```
 
+NOTE: The `default_scope` is also applied while creating/building a record.
+It is not applied while updating a record.
```

Comment:
Is it worth adding an example to the note? Something like:
```ruby
class User < ActiveRecord::Base
  default_scope { where active: true }
end

User.create! name: 'foo'
=> #<User id: 1, name: "foo", active: true>

User.unscoped.create! name: 'bar'
=> #<User id: 2, name: "bar", active: nil>
```

Otherwise, LGTM :smiley: